src/Azure/AzureAD/Authentication.AzureAD.UI/test/Microsoft.AspNetCore.Authentication.AzureAD.UI.Test.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <RootNamespace>Microsoft.AspNetCore.Authentication.AzureAD.UI</RootNamespace>
+    <!-- This package will be deprecated -->
+    <NoWarn>$(NoWarn);CS0618</NoWarn>
```

Comment:
I'm just lazy. Since these are tests for an obsolete package we're going to remove, I didn't want to use pragmas everywhere. I'm not sure if obsolete attribute would interfere with xunit. What's the harm of nowarn? It's a bit of a hammer but in this case, I feel it's not a big deal.

Reply:
Didn't say it was harmful, just didn't look useful because `[Obsolete]` code can use other `[Obsolete]` code IIRC.